.dockerignore

```diff
@@ -0,0 +1,5 @@
+tools/_screenshots
```

Comment:
true, but I don't think it can hurt. the listed files/folders should never be in the docker image, so if they get copied over by accident in future this prevents that

Reply:
Eh, fair enough. 5 dotfiles at the repo root is starting to be a bit much but it can't be helped ¯\_(ツ)_/¯